src/concepts/kafka-proxies/grpc-proxy.md

```diff
@@ -1,297 +1,51 @@
 ---
-description: This guide will walk through each unique gRPC message request and response design and how Zilla is configured to manage the connection for each.
+description: The Zilla gRPC Kafka Proxy lets you implement gRPC service definitions from protobuf files to consume and produce messages from Kafka topics.
 prev: false
 next: /tutorials/grpc/grpc-intro.md
 ---
 
 # gRPC Kafka Proxy
 
-This guide will walk through each unique gRPC message request and response design and how Zilla is configured to manage the connection for each.
+The Zilla gRPC Kafka Proxy lets you implement gRPC service definitions from protobuf files to consume and produce messages from Kafka topics.
```

Comment:
```suggestion
The Zilla gRPC Kafka Proxy lets you implement gRPC service definitions from protobuf files to produce and consume messages via Kafka topics.
```
seems more logical to produce before consume, especially for request-response, and via is neutral to both directions, whereas to and from would be direction-specific